app/controllers/event.js

```diff
@@ -0,0 +1,138 @@
+const Event = require('../models/Event')
+module.exports = {
+  createEvent: async (req, res, next) => {
+    const event = new Event(req.body)
+    try {
+      await event.save()
+      res.status(201).json({ event: event })
+    } catch (error) {
+      console.log(error)
+      res.status(400).json({ error: error })
+    }
+  },
+  updateEvent: async (req, res) => {
+    Event.findById(req.params.id, (err, data) => {
+      if (err) {
+        throw err
```

Comment:
It's good to throw a clear message instead of throwing the whole error object

Reply:
@Rupeshiya , made some suggested changes have a look please